cmd/vl3-nse/vl3_connect.go

```diff
@@ -178,6 +178,7 @@ func (vxc *vL3ConnectComposite) Request(ctx context.Context,
 
 	if vl3SrcEndpointName, ok := conn.GetLabels()[LABEL_NSESOURCE]; ok {
 		// request is from another vl3 NSE
+		conn.Labels["ucnf/peerName"] = vl3SrcEndpointName
```

Comment:
I think we should add `ucnf/peerName` as a const in "github.com/cisco-app-networking/nsm-nse/pkg/universal-cnf/config"